plugins/catalog-backend-extension-msgraph/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@backstage/plugin-catalog-backend-extension-msgraph",
```

Comment:
`@backstage/plugin-catalog-backend-module-msgraph`

:shipit: ?

Reply:
I'm open to a different name 😉 Maybe we even want to update [ADR011](https://backstage.io/docs/architecture-decisions/adrs-adr011) with the new naming pattern for extensions to existing plugins.

Reply:
Yep it should ultimately go into the ADR I think.

We're in intense discussions about this thing and its implications :)

Reply:
Maybe even without a prefix at all? `@backstage/plugin-catalog-backend-msgraph`. I think `module` is a quite overloaded name in the JS ecosystem. On the other hand, `extension` is already used in Backstage for other stuff. Even more funny the name would contain `plugin` and `extension` which is more or less the same 😅

Reply:
Yeah that was a reason to consider abandoning "extension" because of similarity to "plugin".

Having nothing at all could work, but means a risk of collision with `-node` etc, things that we want to add in the future. Essentially that "slot" in the name might be good to be controllable and enumerable by us.

Reply:
Yeah, collisions could be a problem... So `module` then? I will rename it.

Reply:
It's what we voted for at least :)

Reply:
Here we go, a new name. Also removed the example code, should be ready now.